core/lib/external_price_api/src/coingecko_api.rs

```diff
@@ -110,3 +132,38 @@ impl CoinGeckoPriceResponse {
             .and_then(|price| price.get(currency))
     }
 }
+mod test {
+    use std::str::FromStr;
+
+    use zksync_config::configs::ExternalPriceApiClientConfig;
+    use zksync_types::Address;
+
+    use super::CoinGeckoPriceAPIClient;
+
+    #[tokio::test]
+    // This is not a proper test, as it requires user input and prints information.
+    // It serves as a quick check to verify if the CoinGecko api_key is working correctly.
+    async fn test_coingecko_api() {
+        let config = ExternalPriceApiClientConfig {
+            source: "coingecko".to_string(),
+            base_url: Some("https://api.coingecko.com".to_string()),
+            api_key: Some("insert_your_api".to_string()),
+            client_timeout_ms: 30000,
+            forced_numerator: Some(100),
+            forced_denominator: Some(1),
+            // If the below address is set the `get_token_price_by_address` will ommit the address passed as parameter
+            //forced_ethereum_erc20_address: Some("0xdac17f958d2ee523a2206206994597c13d831ec7".to_string()),
+            forced_ethereum_erc20_address: None,
+        };
+        let api = CoinGeckoPriceAPIClient::new(config);
+
+        println!(
+            "{:?}",
+            api.get_token_price_by_address(
+                Address::from_str("0x1f9840a85d5af5bf1d1762f925bdaddc4201f984").unwrap()
+            )
+            .await
+            .unwrap()
```

Comment:
General observation: such tests are prone to be very flaky because the availability of 3rd party API is not predictable.
If we want to keep it, it's better to mark it as `ignore` so that it only runs if `--ignored` flag is provided.

Reply:
Fixed!